control-plane-operator/controllers/hostedcontrolplane/cloud/kubevirt/providerconfig.go

```diff
@@ -39,7 +39,7 @@ func (c *CloudConfig) serialize() (string, error) {
 	return string(out), nil
 }
 
-func cloudConfig(namespace string) CloudConfig {
+func cloudConfig(namespace string, kubeconfig string) CloudConfig {
```

Comment:
maybe call `kubeconfig` something like `kubeconfigPath` so it's obvious we're dealing with a path here.